pekko-sample-persistence-dc-scala/src/main/resources/application.conf

```diff
@@ -52,8 +52,8 @@ akka.persistence.cassandra {
 datastax-java-driver.advanced.reconnect-on-init = true
 
 
-# Akka Management config: https://developer.lightbend.com/docs/akka-management/current/index.html
-akka.management {
+# Apache Pekko Management config: https://developer.lightbend.com/docs/akka-management/current/index.html
+pekko.management {
   http.hostname = "127.0.0.1"
   http.port = 19999
```

Comment:
Question: Is this port correct?

Reply:
let's just keep this value, for now.

Seems the default in Akka management is 8558. We haven't yet changed the Pekko equivalent.